tools/Utils/c_rehash.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python
+
+import os
+import argparse
+import subprocess
+
+def process_file (dirname, file) :
+    # Old full file ocation
+    full_path = dirname + "\\" + file
+    # Calculate hash using "OpenSSL x509" functionality
+    cmd = subprocess.Popen('openssl x509 -hash -noout -in ' + full_path, shell=True, stdout=subprocess.PIPE)
+    for hash in cmd.stdout:
+        hahs_filename = hash.rstrip('\n') + ".0"
+        print file + " => " + hahs_filename
+        # New cert name with hash 
+        new_path = dirname + "\\" + hahs_filename
+        subprocess.call('copy ' + full_path + " " + new_path + " >NUL", shell=True)
+
+def process_dir (dir) :
+    files = os.listdir(dir)
+    for filename in files:
+        if (".crt" in filename) and not (".req" in filename) :
```

Comment:
@dev-gh , no. I just need to created some like symlinks named with hash of cert. (only .crt)
About .req : our generator creates .crt.req files next to .crt , i dont know why. I just filter it.